Assets/Scenes/TestScene.unity

```diff
@@ -2386,7 +2386,107 @@ Prefab:
       objectReference: {fileID: 0}
     - target: {fileID: 1860666892717170, guid: 30eb4c1a1bd224c44a0d4ac296483ba9, type: 2}
       propertyPath: m_IsActive
-      value: 0
```

Comment:
Please discard the changes to this scene unless you made major, necessary changes.